sql/src/main/java/io/crate/execution/engine/aggregation/GroupBySingleNumberCollector.java

```diff
@@ -82,17 +95,6 @@
         this.accumulator = mode == AggregateMode.PARTIAL_FINAL ? this::reduce : this::iter;
     }
 
-    private static void validateInputType(DataType valueType) {
```

Comment:
Removed this because there is a validation already here https://github.com/crate/crate/pull/8773/files#diff-a86cfcc82d27706e40a95e3d09e24d60R177